### PR TITLE
Implemented new design using latest o-colors and o-typography

### PR DESCRIPTION
**Pull request by JakeChampion**
I'm still working on the migration guide, opening pull request to get feedback on what has been done.

Old:
0 0 0 0-8081-demos-local-pa11y html
New:
www ft com-__origami-service-build-v2-demos-o-forms 3 5 0-pa11y

### CODE REVIEW

File: src/scss/mixins/_radio-checkbox.scss [hunk 1]
```diff
@@ -114,9 +117,10 @@
 	}
 
 	#{$checkbox} + #{$label}::after {
```
Comment (olipowell) [inline src/scss/mixins/_radio-checkbox.scss]: Update to new tick icon, and increase background-size: 40px

File: src/scss/mixins/_common.scss [hunk 2]
```diff
@@ -60,6 +63,8 @@
 /// Common field styles: valid state
 @mixin oFormsCommonFieldValid {
 	border-color: oColorsGetColorFor(o-forms-field-valid, border);
+	background-color: rgba(oColorsGetColorFor(o-forms-field-valid, background), 0.05);
+	color: oColorsGetColorFor(o-forms-field-valid, text);
 }
 
 /// Common field styles: small
```
Comment (JakeChampion) [inline src/scss/mixins/_common.scss]: Done.

File: src/scss/mixins/_select.scss [hunk 3]
```diff
@@ -2,9 +2,9 @@
 ///
 /// To be used in combination with oFormsCommonField
 @mixin oFormsSelect {
-	$icon-color: oColorsGetPaletteColor('grey-tint5');
+	$icon-color: oColorsGetPaletteColor('black-80');
 	cursor: pointer;
-	color: oColorsGetPaletteColor('grey-tint5');
+	color: oColorsGetPaletteColor('black-80');
 	background-image: url(str-replace("https://www.ft.com/__origami/service/image/v2/images/raw/fticon-v1:arrows-up-down?source=o-icons&tint=%23#{$icon-color}&format=svg", "#", ""));
```
Comment (JakeChampion) [inline src/scss/mixins/_select.scss]: Done.

File: src/scss/prefix-suffix.scss [hunk 4]
```diff
@@ -3,6 +3,13 @@
 		display: table;
 		width: 100%;
 		box-sizing: border-box;
+		@include oTypographyMargin($top: 1);
+	}
+
```
Comment (JakeChampion) [inline src/scss/prefix-suffix.scss]: Done.

File: src/scss/mixins/_radio-checkbox.scss (hunk 1 above)
Comment (JakeChampion) [inline src/scss/mixins/_radio-checkbox.scss]: Waiting on the tick icon to be updated 👍

Comment (JakeChampion) [inline src/scss/mixins/_radio-checkbox.scss]: PR for the new tick icon -- https://github.com/Financial-Times/fticons/pull/33

**Pull request merged**